scipy/linalg/flapack_64.pyf.src

```diff
@@ -16,9 +16,23 @@
 
 python module _flapack_64
     usercode '''
-#if defined(BLAS_SYMBOL_PREFIX) || defined(BLAS_SYMBOL_SUFFIX)
-#include "blas64-prefix-defines.h"
```

Comment:
xref https://github.com/scipy/scipy/pull/19816#issuecomment-1879792156 for what this used to do

(no change to the code here needed)

Reply:
That said, the lines below are now repeated twice, for `fblas_64` and `flapack_64` - would be good to put that into a new header.